packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+    title="Empty States with immediate next steps"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+### Empty state requiring a customer to wait
+
+A customer's next step might not always require an immediate call-to-action. Instead, the best path forward for them might be to wait for another process to complete and come back later. This happens most often when a customer is awaiting data through event logs, or when another person or non-Twilio service needs to approve a customer's request.
+
+If the customer might need to start another workflow, consider giving them a way back to a dashboard or the start of another flow.
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-requiring-a-customer-to-wait-uyxu3?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+### Sizing & Responsiveness
+
+When an empty state card is wider than 540px (recommended), use `padding="space150"`, and align illustrations horizontally with the text content.
+
+In smaller spaces, use `padding="space100"` and stack the illustration in the card above the text content.
+
+If your empty state composition needs to accommodate different screen sizes, you can also wrap it in the Stack or Grid component.
+
+<Callout>
+  <CalloutText>
+    If you'd like to contribute guidelines for empty states in even more compact situations, like a navigation panel or
+    components like <Anchor href="/components/combobox">Combobox</Anchor>, please reach out in a{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions">GitHub Discussion</Anchor>!
+  </CalloutText>
+</Callout>
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-sizing-responsiveness-euj5p?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+## Content examples by use case
```

Comment:
This heading was hard for me to understand. Maybe we could switch it to something like "Examples of Empty States"?

Reply:
hm, true. Let's go with "Examples of Empty State use cases"